src/app/server/java/AndroidAppServerWrapper.cpp

```diff
@@ -62,3 +62,11 @@ CHIP_ERROR ChipAndroidAppInit(void)
     }
     return err;
 }
+
+CHIP_ERROR ChipAndroidAppShutdown(void)
```

Comment:
Since this never returns anything except `NO_ERROR` is there value in it having a return type other than void?